app/store/index.js

```diff
@@ -2,10 +2,12 @@ import { action, observable } from "mobx";
 import contract from "truffle-contract";
 import AuctionBaseContract from "../../build/contracts/AuctionBase.json";
 import HillCoreContract from "../../build/contracts/HillCore.json";
+import { HILL_CORE_ABI, HILL_CORE_ADDRESS } from "../ERC821Contracts";
```

Comment:
It'd be nicer to do `import { HillCore } from "../contracts/erc821"` and then be able to reference `HillCore.abi` and `HillCore.address`. Can you make that change?

Reply:
Fixed